src/app/admin/backup.ts

```diff
@@ -63,7 +63,13 @@ export const uploadBackup =
             addEventToCurrentSpan("Static channel backup to Dropbox successful.")
           } catch (error) {
             logger.error({ error }, "Static channel backup to Dropbox failed.")
-            recordExceptionInCurrentSpan({ error: error, level: ErrorLevel.Warn })
+            if (error instanceof Error)
+              recordExceptionInCurrentSpan({ error: error, level: ErrorLevel.Warn })
+            if (typeof error === "string")
+              recordExceptionInCurrentSpan({
+                error: new Error(error),
+                level: ErrorLevel.Warn,
+              })
```

Comment:
do we need this branch?

Reply:
you are right, axios should not return a string